pyproject.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
   "typer==0.9.0",
   "urllib3<1.27,>=1.21.1",
   "GitPython==3.1.37",
+  "pyyaml==6.0.1",
```

Comment:
Please use `strictyml.load` instead of adding a new dependency.

Reply:
addressed comment